sdk/eventhub/eventhubs-checkpointstore-blob/sample.env

```diff
@@ -0,0 +1,17 @@
+# Used in most samples
```

Comment:
`STORAGE_CONNECTION_STRING=<connection string for an Azure Storage account>`

(this wasn't documented but it is needed for the test suite).

Reply:
I'll follow up with this later today.